.github/workflows/release.yml

```diff
@@ -10,9 +10,17 @@ jobs:
   build:
     strategy:
       matrix:
-        operating-system: [ubuntu-18.04, windows-2019, macos-10.15]
```

Comment:
https://github.community/t/create-matrix-with-multiple-os-and-env-for-each-one/16895